test/Microsoft.Azure.Devices.Edge.Test/EdgeAgentDirectMethods.cs

```diff
@@ -68,6 +69,43 @@ public async Task TestGetModuleLogs()
             string expected = string.Join('\n', Enumerable.Range(0, count)) + "\n";
             LogResponse response = JsonConvert.DeserializeObject<LogResponse[]>(result.GetPayloadAsJson()).Single();
             Assert.AreEqual(expected, response.Payload);
+
+            // Verify Unix Time Operation
```

Comment:
This adds more moving parts to the existing test (and everything still has to complete within the alloted 5 min limit for tests), so we'll need to keep a close eye on stability here. I'm thinking it might be better to test _duration only_ in this test, since that's what the portal uses. Then we can rely on the edgelet unit tests to make sure all the other (RFC 3339, unix timestamp) formats work as expected.

Reply:
I just observed the timestamps on the E2E Test logs and noticed that all 4 requests were returned within a second of each other. Do you think the behavior might vary a lot?

```
[09:09:19 VRB] Method 'GetModuleLogs' on 'e2e-7c801001001-211116-090809.297-001/$edgeAgent' returned: 200
[{"id":"NumberLogger","payloadBytes":null,"payload":"0\n1\n2\n3\n4\n5\n6\n7\n8\n9\n"}]
[09:09:19 VRB] Method 'GetModuleLogs' on 'e2e-7c801001001-211116-090809.297-001/$edgeAgent' returned: 200
[{"id":"NumberLogger","payloadBytes":null,"payload":"0\n1\n2\n3\n4\n5\n6\n7\n8\n9\n"}]
[09:09:19 VRB] Method 'GetModuleLogs' on 'e2e-7c801001001-211116-090809.297-001/$edgeAgent' returned: 200
[{"id":"NumberLogger","payloadBytes":null,"payload":"0\n1\n2\n3\n4\n5\n6\n7\n8\n9\n"}]
[09:09:19 VRB] Method 'GetModuleLogs' on 'e2e-7c801001001-211116-090809.297-001/$edgeAgent' returned: 400
```

Reply:
Oh, ok. I guess since we're just requesting logs for NumberLogger, and that module has very (intentionally) minimal logs, these are not expensive calls. We should be ok making requests using all time formats in this test. ✅